### DISCUSSION TITLE
Lack of documentation on user configuration file

### DISCUSSION THREAD

**Discussion by t-wissmann** [davatorium/rofi#1432] [(Possible) Issues]
The [section `CONFIGURATION` in the man page](https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#configuration) does not mention (the format of) the user's configuration file in `XDG_CONFIG_HOME`/`XDG_CONFIG_DIRS`. Currently, it only says:

> System configuration file (for example /etc/rofi.rasi). It first checks `XDG_CONFIG_DIRS`, and then `SYSCONFDIR` (that is passed at compile time). It loads the first config file it finds, it does not merge multiple system configuration files.

Instead, I would expected a man page to additionally specify something along the lines of »it first checks `rofi/config.rasi` in one of the `XDG_CONFIG_DIRS`«. Another option is to say something like: »see the output of `rofi -h` for the concrete path of the configuration file«. (Both formulations are only suggestions, but I think the concrete suffix of the filepath should be explicitly mentioned in the `CONFIGURATION` section of the man page)

Also the backup filepath `~/.config/rofi/config.rasi` is not mentioned (in fact, even the string `config/rofi` is not mentioned in the entire man page)

I've already talked to @dannycolin on the irc, and I was asked to open this ticket here.

**Comment by DaveDavenport** [comment]
If anybody is willing to help with documentation this would be much appreciated.

**Reply by dannycolin** [reply]
@DaveDavenport I can have a look this week.
